docs/src/components/Header/Header.astro

```diff
@@ -0,0 +1,155 @@
+---
+import SkipToContent from './SkipToContent.astro';
+import SidebarToggle from './SidebarToggle.tsx';
+import LanguageSelect from './LanguageSelect.jsx';
+import Search from "./Search.jsx";
+---
+<style>
+      header {
+        z-index: 11;
+        height: var(--theme-navbar-height);
+        width: 100%;
+        background-color: var(--theme-navbar-bg);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        overflow: hidden;
+        position: sticky;
+        top: 0;
+      }
+
+
+    .logo {
+    display: flex;
+    overflow: hidden;
+        width: 30px;
+    font-size: 1rem;
+    flex-shrink: 0;
+    font-weight: 600;
+    line-height: 1;
+    color: hsla(var(--color-base-white), 100%, 1);
+    text-decoration: none;
+    gap: 0.5em;
+    z-index: -1;
+    }
+
+    .logo a {
+    padding: 0.5em 0.25em;
+    margin: -0.5em -0.25em;
+    }
+
+    .logo svg {
+    height: 40px;
+    width: auto;
+    display: block;
+    color: var(--theme-accent);
+    }
+
+    .logo .hover {
+    opacity: 0.0;
+    }
+    .logo a {
+      transition: transform 180ms ease-out;
+    }
+
+    .logo a:hover,
+    .logo a:focus {
+    outline: none;
+    opacity: 1.0;
+    transform: translateY(-2px);
+    }
+
+    .logo h1 {
+    font: inherit;
+    color: inherit;
+    margin: 0;
+    }
+
+
+
+    .nav-wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    gap: 1em;
+    width: 100%;
+    max-width: 82em;
+    padding: 0 1rem;
+    }
+    @media (min-width: 50em) {
+        header {
+            position: static;
+            padding: 2rem 0rem 0 2rem;
+        }
+        .logo {
+            width: auto;
+            margin: 0;
+            z-index: 0;
+        }
+        .menu-toggle {
+            display: none;
+        }
+        .logo {
+            width: auto;
+        }
+    }
+
+    /** Style Algolia */
+:root {
+  --docsearch-primary-color: var(--theme-accent);
+  --docsearch-logo-color: var(--theme-text);
+}
+
+.search-item {
+  display: none;
+  position: relative;
+  z-index: 10;
+  flex-grow: 1;
+  padding-right: 0.7rem;
+  display: flex;
+  max-width: 200px;
+}
+:global(.search-item > *) {
+  flex-grow: 1;
+}
+@media (min-width: 50em) {
+  .search-item {
+    max-width: 400px;
+  }
+}
+</style>
+<header>
+    <SkipToContent />
+    <nav class="nav-wrapper" title="Top Navigation">
+    <div class="menu-toggle">
+        <SidebarToggle client:idle/>
```

Comment:
The sidebar toggle doesn't have active or hover styles anymore, maybe could the global button design be changed?